src/backend/globals.py

```diff
@@ -44,6 +44,16 @@ def get_apps_dir():
         _dir = f"{Path.home()}/.local/share/applications"
     return _dir
 
+def VkBasaltPath():
+    vkbasalt_paths = [
+        "/usr/lib/extensions/vulkan/vkBasalt/share/vulkan/implicit_layer.d/vkBasalt.json",
+        "/usr/share/vulkan/implicit_layer.d/vkBasalt.json"
+    ]
+    for path in range(len(vkbasalt_paths)):
+        if os.path.isfile(vkbasalt_paths[path]):
```

Comment:
You can simply return this as os.path.file already returns a bool

Reply:
Doing that will only return the first path no? I committed https://github.com/bottlesdevs/Bottles/pull/1668/commits/057d30f01a3b6d83d82a8d4a716924ad569342b1 so it returns False only if those directories are missing.

Reply:
'os.path.isfile` always return a bool:
```python
>>> import os
>>> print(os.path.isfile("this/path/doesnt/exists"))
False
>>> print(os.path.isfile("/var/home/mirko/this_exists"))
True
>>> 
```

Reply:
That I understand, but what the script is trying to do is check for files in a given list. Your method seems to not work with lists:
```py
>>> import os
>>> install_paths = [ "/usr/lib/extensions/vulkan/vkBasalt/share/vulkan/implicit_layer.d/vkBasalt.json", "/usr/share/vulkan/implicit_layer.d/vkBasalt.json" ]
>>> if os.path.isfile(install_paths):
...     print("Yes")
... 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.9/genericpath.py", line 30, in isfile
    st = os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not list
```

Reply:
You need to keep the loop, just change the return

Reply:
for …:
    return os.path.isfile(..)

instead of
for ..:
    if os.path.isfile(..):
        return True
    return False

Reply:
So something like this?
```py
    vkbasalt_paths = [
        "/usr/lib/extensions/vulkan/vkBasalt/share/vulkan/implicit_layer.d/vkBasalt.json",
        "/usr/share/vulkan/implicit_layer.d/vkBasalt.json"
    ]
    for path in range(len(vkbasalt_paths)):
        return os.path.isfile(vkbasalt_paths[path])
```

Reply:
Won't that only return the very first file?

Reply:
Isn’t already doing it? You are returning inside the loop

Reply:
That was fixed https://github.com/bottlesdevs/Bottles/pull/1668/commits/057d30f01a3b6d83d82a8d4a716924ad569342b1.

Reply:
Correction: I need to fix it :p

Reply:
Correction, I need to fix it :p

Reply:
Fixed. https://github.com/bottlesdevs/Bottles/pull/1668/commits/9111090e0d0f7701050e0f88f0c291f3e8ecabc9